src/schema/v2/previewSavedSearch.ts

```diff
@@ -27,6 +27,9 @@ const previewSavedSearchArgs: GraphQLFieldConfigArgumentMap = {
   artistIDs: {
     type: new GraphQLList(GraphQLString),
   },
+  artistSeriesIDs: {
+    type: new GraphQLList(GraphQLString),
+  },
```

Comment:
@nickskalkin it's possible your Eigen work is blocked by this argument not being recognized, but this should take care of it.

Reply:
yes, I've drafted a PR (https://github.com/artsy/eigen/pull/9558) on Friday, but tests fail and input name is also not shown.